src/V3Split.cpp

```diff
@@ -881,12 +945,73 @@ class SplitVisitor final : public SplitReorderBaseVisitor {
     // METHODS
 protected:
     virtual void makeRvalueEdges(SplitVarStdVertex* vstdp) override {
+        AstVarScope* const vscp = VN_CAST(vstdp->nodep(), VarScope);
+        UASSERT_OBJ(vscp, vstdp->nodep(), "RHS should be VarScope");
+        auto it = m_lhsRevision.find(vscp);
+        const int rev = it == m_lhsRevision.end() ? 0 : it->second;
         // Each 'if' depends on rvalues in its own conditional ONLY,
         // not rvalues in the if/else bodies.
         for (auto it = m_stmtStackps.cbegin(); it != m_stmtStackps.cend(); ++it) {
             const AstNodeIf* const ifNodep = VN_CAST((*it)->nodep(), NodeIf);
             if (ifNodep && (m_curIfConditional != ifNodep)) continue;
-            new SplitRVEdge(&m_graph, *it, vstdp);
+            new SplitRVEdge(&m_graph, *it, vstdp, rev);
```

Comment:
```suggestion
            new SplitRVEdge{&m_graph, *it, vstdp, rev};
```

(New code should upgrade to {} constuctors.)